src/common/PreviewUtils.ts

```diff
@@ -68,4 +72,15 @@ export class PreviewUtils {
         const configFile = Object.assign(new PreviewConfigFile(), json);
         return configFile;
     }
+
+    public static getAppBundlePath(
+        appConfig: IOSAppPreviewConfig | AndroidAppPreviewConfig
+    ): string | undefined {
+        if (appConfig.get_app_bundle) {
+            const module = require(appConfig.get_app_bundle);
```

Comment:
We have a requirement in the spec for relative paths that states, "If not absolute, the path to the module should be relative to the configuration file." The place where we're going to need this is when we do an "internal" app configuration file for test harness apps, and we're going to need to specify a relative path for the get_app_bundle modules, because we won't have access to the absolute path.

Reply:
oh good catch, forgot about that. Will update.